.github/workflows/build.yml

```diff
@@ -16,8 +16,13 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["7.3", "7.4", "8.0"]
-                symfony: ["4.4.*", "5.1.*"]
+                php: ["7.3", "7.4"]
+                symfony: ["5.3.*", "5.4.0"]
```

Comment:
Why did you removed Sf5.3 & PHP8.0 build?

Reply:
This is below :)

Reply:
OOPs you're right!

Reply:
@lchrusciel Fixed